### PR TITLE
Improve readability of the Sepia page

### PR DESCRIPTION
**Pull request by m-linner-ericsson**
### Applicable Issues
An attempt to increase the readability of the Sepia architecture page. 

### Description of the Change
A preview of the page can be found on: https://m-linner-ericsson.github.io/eiffel-sepia/index.html

- Use [Chota](https://jenil.github.io/chota/) as base for CSS. This will:
	- Limit the width of the page to about 1200 (same as this page)
	- Enable use of nav element
	- Easier to port to other front-end frameworks if needed
- All links have the same color
- Visited links are different color
- External links are denoted with an icon
- Grey backgrounds used in text removed (hard to see links in these sections)

### Alternate Designs
I tried to use the smallest possible framework. One could go with e.g. [Twitter Bootstrap](https://getbootstrap.com/) but that felt too much for now

### Benefits
Easier to read and makes it easier to port to other front-end frameworks

### Possible Drawbacks
The pages don't look exactly the same as before and do not match the style of the community pages. (but will when https://github.com/eiffel-community/eiffel-community.github.io/pull/21 get merged)

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Mattias Linnér mattias.linner@ericsson.com

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -270,81 +47,6 @@ section {
...
-    font-size: 24px;
-  }
-
-  .splash-text-small {
-    font-size: 12px;
-  }
-}
+svg {
+    padding-bottom: 50px;
+}
```
Comment (magnusbaeck) [inline css/index.css]: Nit: File should end with a newline character.

File: includes/header.html [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
+            </a>
+            <a class="no-shrink" href="https://github.com/eiffel-community/eiffel-sepia">
+              <img src="images/github-organization-icon.png" class="header-nav-link-image"/>
+            </a>
+            <a class="no-shrink" href="https://groups.google.com/forum/#!forum/eiffel-community">
+              <img src="images/google-group-icon.png" class="header-nav-link-image"/>
+            </a>
+        </div>
+      </nav>
+    </header>
```
Comment (magnusbaeck) [inline includes/header.html]: Nit: File should end with a newline character.

**Review by magnusbaeck — APPROVED** [review]
Good job, this is big improvement over the current look! I hope we can apply the same changes to https://eiffel-community.github.io/.

My CSS fu is limited so I'm mostly reviewing the end result.

File: css/index.css (hunk 1 above)
Comment (m-linner-ericsson) [reply css/index.css]: Fixed in 71136f8

File: includes/header.html (hunk 2 above)
Comment (m-linner-ericsson) [reply includes/header.html]: Fixed in 71136f8

**Review by e-backmark-ericsson — APPROVED** [review]

**Pull request merged**